CMakeLists.txt

```diff
@@ -573,128 +570,33 @@ if(QUDA_QDPJIT)
 endif()
 
 if(QUDA_QMP)
-  find_library(QMP_FOUND qmp HINTS ${QUDA_QMPHOME}/lib NO_DEFAULT_PATH)
-  mark_as_advanced(QMP_FOUND)
-  set_property(DIRECTORY APPEND PROPERTY CMAKE_CONFIGURE_DEPENDS ${QMP_FOUND})
-  if(QUDA_DOWNLOAD_USQCD AND NOT QMP_FOUND)
-    ExternalProject_Add(
-      QMP
-      GIT_REPOSITORY https://github.com/usqcd-software/qmp.git
-      GIT_TAG qmp2-5-2
-      GIT_SHALLOW YES
-      PREFIX usqcd
-      CONFIGURE_COMMAND CC=${MPI_C_COMPILER} CXX=${MPI_CXX_COMPILER} <SOURCE_DIR>/configure "INSTALL=${INSTALL_EXE} -C"
-                        "CFLAGS=-Wall -O3 -std=c99 " --with-qmp-comms-type=MPI --prefix=<INSTALL_DIR>
-      BUILD_COMMAND ${MAKE_EXE}
-      INSTALL_COMMAND ${MAKE_EXE} install
-      LOG_INSTALL ON
-      LOG_BUILD ON
-      LOG_DOWNLOAD ON)
-
-    ExternalProject_Get_Property(QMP INSTALL_DIR)
-    set_property(DIRECTORY APPEND PROPERTY CMAKE_CONFIGURE_DEPENDS ${INSTALL_DIR})
-    set(QUDA_QMPHOME
-        ${INSTALL_DIR}
-        CACHE PATH "path to QMP" FORCE)
-    set(QUDA_QMP_LDFLAGS
-        "-L${QUDA_QMPHOME}/lib"
-        CACHE STRING "LDFLAGS for QMP - should be derived from qmp-config --ldflags")
-    set(QUDA_QMP_LIBS
-        "-lqmp"
-        CACHE STRING "LIBS for QMP - should be derived from qmp-config --libs")
-    ExternalProject_Add_Step(
-      QMP reconf
-      COMMAND ${AUTORECONF_EXE} -fi
-      WORKING_DIRECTORY <SOURCE_DIR>
-      DEPENDERS configure
-      DEPENDEES download)
-
+  message(STATUS "Looking for QMP")
```

Comment:
This is the modern cmake way to look for a QMPConfig.cmake. One either needs to set QMP_DIR to <install_path_for_QMP>/lib/cmake/QMP 
or add <install_path_for_QMP> to CMAKE_PREFIX_PATH